conf/vertebrates/jira_recurrent_tickets.json

```
@@ -342,5 +342,24 @@
          }
       ],
       "summary": "<Division> Release <version> Final checks before public release"
+   },
+   {
+      "assignee": "<RelCo>",
+      "component": "Relco tasks",
+      "description": "https://www.ebi.ac.uk/seqdb/confluence/display/EnsCom/Flat+file+dumps",
+      "subtasks": [{
+            "assignee": "<RelCo>",
+            "component": "Production tasks",
+            "description": "https://www.ebi.ac.uk/seqdb/confluence/display/EnsCom/Flat+file+dumps#Flatfiledumps- Databaseupdate",
```

Not the end of the world to leave it as it is, but removing the space from the link will save a moment for the Vertebrates relco:
```suggestion
            "description": "https://www.ebi.ac.uk/seqdb/confluence/display/EnsCom/Flat+file+dumps#Flatfiledumps-Databaseupdate",
```